src/broker/Manager.cc

```diff
@@ -1582,7 +1582,14 @@ void Manager::ProcessStoreResponse(detail::StoreHandleVal* s, broker::store::res
 }
 
 detail::StoreHandleVal* Manager::MakeMaster(const string& name, broker::backend type, broker::backend_options opts) {
-    if ( bstate->endpoint.is_shutdown() )
+    if ( ! bstate ) {
```

Comment:
Depending how much we want to do with broker stores, could also queue an re-run `MakeMaster` once `bstate` was initialized?

Reply:
Would that do anything for the example given in the ticket? `MakeMaster` gets queued, but then we try to use the result of it to create a table. At the time the table gets created, the store handle is still invalid because `MakeMaster` didn't return anything. So now you not only need to queue the creation, but also the table creation.

Reply:
yeah, just scrolling a bit, this was a very naive suggestion.